### PR TITLE
Add changelog for 0.7.0 release

### PR DESCRIPTION
**Pull request by saad-ali**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:

/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

/assign @msau42

### CODE REVIEW

File: CHANGELOG-0.7.0.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Changelog since v0.6.1
+
```
Comment (msau42) [inline CHANGELOG-0.7.0.md]: Can you also add a Breaking Changes section, that points to the 2 changes that modified the existing interfaces?

File: CHANGELOG-0.7.0.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+## New Features
+
+- Introduce a CSI Metrics Library that can be used to automatically record prometheus metrics for every CSI gRPC call. ([#35](https://github.com/kubernetes-csi/csi-lib-utils/pull/35), [@saad-ali](https://github.com/saad-ali))
+
+
+## Other Notable Changes
```
Comment (msau42) [inline CHANGELOG-0.7.0.md]: Also add a note that k8s dependencies have been updated to 1.17.0

File: CHANGELOG-0.7.0.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Changelog since v0.6.1
+
+## New Features
+
+- Introduce a CSI Metrics Library that can be used to automatically record prometheus metrics for every CSI gRPC call. ([#35](https://github.com/kubernetes-csi/csi-lib-utils/pull/35), [@saad-ali](https://github.com/saad-ali))
```
Comment (msau42) [inline CHANGELOG-0.7.0.md]: Also include the new package name

File: CHANGELOG-0.7.0.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Changelog since v0.6.1
```
Comment (msau42) [inline CHANGELOG-0.7.0.md]: Also can you rename the filename to the minor release? CHANGELOG-0.7.md

File: CHANGELOG-0.7.0.md (hunk 2 above)
Comment (saad-ali) [reply CHANGELOG-0.7.0.md]: Done.

File: CHANGELOG-0.7.0.md (hunk 3 above)
Comment (saad-ali) [reply CHANGELOG-0.7.0.md]: Done.

File: CHANGELOG-0.7.0.md (hunk 4 above)
Comment (saad-ali) [reply CHANGELOG-0.7.0.md]: Done

File: CHANGELOG-0.7.0.md (hunk 1 above)
Comment (saad-ali) [reply CHANGELOG-0.7.0.md]: Done

**Comment by saad-ali** [comment]
Feedback addressed. PTAL

**Pull request merged**